scripts/deploy_dag.py

```diff
@@ -14,14 +14,24 @@
 
 
 import argparse
+import json
 import pathlib
 import subprocess
 import typing
 import warnings
 
+from ruamel import yaml
+
+yaml = yaml.YAML(typ="safe")
+
 CURRENT_PATH = pathlib.Path(__file__).resolve().parent
 PROJECT_ROOT = CURRENT_PATH.parent
 DATASETS_PATH = PROJECT_ROOT / "datasets"
+DEFAULT_AIRFLOW_VERSION = 1
```

Comment:
is this one of the ones that needs to change to 2?

Reply:
Good catch, thank you! 

If you don't mind, I created #138 and I added the correction there instead of here (to prevent possible merge conflicts and re-approving later).

Also as promised in #138, I added the new pipeline.yaml sample that uses Airflow 2 operators in it.